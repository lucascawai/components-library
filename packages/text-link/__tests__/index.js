import React from 'react'
import { TextLink } from '../index'
import { renderWithTheme } from '../../../test/helpers'
import 'jest-styled-components'

describe('TextLink', () => {
  it('renders correctly', () => {
    const component = renderWithTheme(<TextLink text="Test" onClick={jest.fn} color="red" />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
