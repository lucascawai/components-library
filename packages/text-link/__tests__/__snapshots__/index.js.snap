// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextLink renders correctly 1`] = `
.c0 {
  font-family: 'CenturyGothicBold',sans serif;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
  color: red;
}

@media (max-width:800px) {
  .c0 {
    margin-top: 16px 0;
    font-size: 12px;
  }
}

<h5
  className="c0"
  color="red"
  onClick={[Function]}
>
  Test
</h5>
`;
